fe/fe-core/src/test/java/com/starrocks/analysis/CreateRoutineLoadStmtTest.java

```diff
@@ -333,288 +467,656 @@ public void testLoadColumns() {
+            Assert.assertEquals(createRoutineLoadStmt.getEscape(), '|');
+        }
+
+        {
+            String createSQL = "CREATE ROUTINE LOAD db0.routine_load_1 ON t1 " +
+                    "PROPERTIES(\"format\" = \"csv\", \"trim_space\"=\"true\", \"enclose\"=\"'\", \"escape\"=\"|\") " +
+                    "FROM KAFKA(\"kafka_broker_list\" = \"xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx\",\"kafka_topic\" = \"topic_0\");";
+            ConnectContext ctx = starRocksAssert.getCtx();
+            CreateRoutineLoadStmt createRoutineLoadStmt = (CreateRoutineLoadStmt) SqlParser.parse(createSQL, 32).get(0);
+            CreateRoutineLoadAnalyzer.analyze(createRoutineLoadStmt, connectContext);
+            Assert.assertEquals(createRoutineLoadStmt.isTrimspace(), true);
+            Assert.assertEquals(createRoutineLoadStmt.getEnclose(), '\'');
+            Assert.assertEquals(createRoutineLoadStmt.getEscape(), '|');
+        }
     }
 
     @Test
     public void testAnalyzeCSVDefalultValue() throws Exception {
-        String createSQL = "CREATE ROUTINE LOAD db0.routine_load_1 ON t1 " +
-                "PROPERTIES(\"max_error_number\" = \"10\") " +
-                "FROM KAFKA(\"kafka_broker_list\" = \"xxx.xxx.xxx.xxx:xxx\",\"kafka_topic\" = \"topic_0\");";
-        ConnectContext ctx = starRocksAssert.getCtx();
-        CreateRoutineLoadStmt createRoutineLoadStmt = (CreateRoutineLoadStmt) SqlParser.parse(createSQL, 32).get(0);
-        CreateRoutineLoadAnalyzer.analyze(createRoutineLoadStmt, connectContext);
-        Assert.assertEquals(createRoutineLoadStmt.getMaxErrorNum(), 10);
-        Assert.assertEquals(createRoutineLoadStmt.getEnclose(), 0);
-        Assert.assertEquals(createRoutineLoadStmt.getEscape(), 0);
-        Assert.assertEquals(createRoutineLoadStmt.isTrimspace(), false);
+        {
+            String createSQL = "CREATE ROUTINE LOAD db0.routine_load_1 ON t1 " +
+                    "PROPERTIES(\"max_error_number\" = \"10\") " +
+                    "FROM KAFKA(\"kafka_broker_list\" = \"xxx.xxx.xxx.xxx:xxx\",\"kafka_topic\" = \"topic_0\");";
+            ConnectContext ctx = starRocksAssert.getCtx();
+            CreateRoutineLoadStmt createRoutineLoadStmt = (CreateRoutineLoadStmt) SqlParser.parse(createSQL, 32).get(0);
+            CreateRoutineLoadAnalyzer.analyze(createRoutineLoadStmt, connectContext);
+            Assert.assertEquals(createRoutineLoadStmt.getMaxErrorNum(), 10);
+            Assert.assertEquals(createRoutineLoadStmt.getEnclose(), 0);
+            Assert.assertEquals(createRoutineLoadStmt.getEscape(), 0);
+            Assert.assertEquals(createRoutineLoadStmt.isTrimspace(), false);
+        }
+
+        {
+            String createSQL = "CREATE ROUTINE LOAD db0.routine_load_1 ON t1 " +
+                    "PROPERTIES(\"max_error_number\" = \"10\") " +
+                    "FROM KAFKA(\"kafka_broker_list\" = \"xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx:xxxx\",\"kafka_topic\" = \"topic_0\");";
```

Comment:
Should the ipv6 address be wrapped with `[]`?

Reply:
Thank you for your review, I will fix it